release-notes.md

```diff
@@ -1,3 +1,18 @@
+# Release 1.9.6
+
+## What's New
+* none
+
+## Other changes:
+* updated installation config
+* renewed signing cert
+
+## Bugs fixed:
+* none
+
+## Dependency Updates
+* tsdk is updated to v0.15.6
```

Comment:
personally i like to see the csdk called out here too... it's been proven useful in the past to have both in this line